openslides/agenda/static/js/agenda/site.js

```diff
@@ -280,6 +280,15 @@ angular.module('OpenSlidesApp.agenda.site', ['OpenSlidesApp.agenda'])
                 $scope.alert = { type: 'danger', msg: data.detail, show: true };
             });
         };
+        // gets speech duration of selected speaker in seconds
+        $scope.getDuration = function (speaker) {
+            var beginTimestamp = new Date(speaker.begin_time).getTime()
+            var endTimestamp = new Date(speaker.end_time).getTime()
+            // calculate duration in seconds
+            var seconds = Math.floor( (endTimestamp - beginTimestamp) / 1000 );
+            return seconds;
```

Comment:
proposal:
* Do not use the variable seconds but return the result of Math.floor directly
* Remove the spaces after and before the bracket.